nav2_planner/src/planner_server.cpp

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) 2019 Samsung Research America
```

Comment:
This file looks like it was mostly copy / paste / rename of the navfn code, is it really Copyright Samsung?

Reply:
Probably, I'll readd the intel one